app/assets/stylesheets/pages/nodes_list.scss

```diff
@@ -31,6 +31,23 @@
   }
 }
 
+.pending-nodes-container {
```

Comment:
You can remove this and add `.pending-nodes-container` to https://github.com/kubic-project/velum/blob/a8ed51120b3278611730fa645dfd3cee1b5ab39b/app/assets/stylesheets/pages/nodes_list.scss#L17

```sass
.nodes-container, 
.pending-nodes-container
```